### ISSUE TITLE
contacts are not displaying

### ISSUE THREAD

**Issue by akhila0906** [Lmakgae/WApp-Clone#1]
contacts list is not displaying

**Comment by Hgupta02** [comment]
is there any solution to this??

**Comment by amit-purohit** [comment]
> contacts list is not displaying

Did you get any solution?

**Comment by amit-purohit** [comment]
> contacts list is not displaying

did you deploy cloud function to firebase?

**Comment by PavankumarHegde** [comment]
> > contacts list is not displaying
> 
> did you deploy cloud function to firebase?

yes....users data also stored in firebase cloud firestore but still not working
